test_flake8_graphql.py

```diff
@@ -11,32 +11,73 @@ def test_no_query(flake8dir):
     assert result.out_lines == []
 
 
-def test_GQL100_pass_1(flake8dir):
+def test_GQL100_function_pass(flake8dir):
     flake8dir.make_example_py("""
-        import gql
+        def gql(query):
+            return query
 
         query = gql("query { countries { name } }")
     """)
-    result = flake8dir.run_flake8('--select=GQL')
+    result = flake8dir.run_flake8(['--select=GQL'])
```

Comment:
It’s minor, and this isn’t the most obvious place for it, but I think immutability as documentation is useful.